src/Microsoft.AspNet.Mvc.Core/ActionResults/ObjectResult.cs

```diff
@@ -55,10 +63,16 @@ public ObjectResult(object value)
             if (selectedFormatter == null)
             {
                 // No formatter supports this.
+                logger.LogWarning("No output formatter was found to write the response.");
+
                 context.HttpContext.Response.StatusCode = StatusCodes.Status406NotAcceptable;
                 return;
             }
 
+            logger.LogInformation(
```

Comment:
:+1:

Think of it this way. We want logging (at information level) to produce probably 5-6 log statements for a typical request. Assume that MVC's budget is 1 per-request - and we're already using that for the action that was selected. 

This means that unless explicitly approved, everything that's a success case should be verbose by default.